qa/suites/rados/monthrash/ceph.yaml

```diff
@@ -4,6 +4,10 @@ overrides:
       mon:
         mon min osdmap epochs: 25
         paxos service trim min: 5
+        # prune full osdmaps regularly
+        mon osdmap full prune min: 15
+        mon osdmap full prune interval: 2
+        mon osdmap full prune txsize: 2
```

Comment:
also, let's do qa/suites/rados/thrash/thrashers/*, something in rados/basic, and something in rados/verify.

and in teuthology.git's ceph.conf.template, lets' do something like 50 and 5 and 5?  that's a sane-ish default, perhaps!